app/src/main/res/values/colors.xml

```diff
@@ -220,4 +220,7 @@
 
     <!-- Notification Color -->
     <color name="whats_new_notification_color">#00B3F4</color>
+
+    <!-- Static Shortcut Background Color -->
+    <color name="static_shortcut_background">#F5F5F5</color>
```

Comment:
For the shortcuts, is it possible to avoid using `png` files by using a `<layer-list>` drawable with the foreground and background? See the "Take a shortcut" section [in this article](https://medium.com/androiddevelopers/implementing-adaptive-icons-1e4d1795470e)

Reply:
I changed it using `layer-list`, thanks!